tests/ssl-opt.sh

```diff
@@ -10515,6 +10517,7 @@ run_test    "TLS 1.3: Server side check - mbedtls" \
             -s "tls13 server state: MBEDTLS_SSL_CLIENT_HELLO" \
             -s "tls13 server state: MBEDTLS_SSL_SERVER_HELLO" \
             -s "tls13 server state: MBEDTLS_SSL_ENCRYPTED_EXTENSIONS" \
+            -s "tls13 server state: MBEDTLS_SSL_SERVER_CERTIFICATE" \
             -c "client state: MBEDTLS_SSL_ENCRYPTED_EXTENSIONS" \
```

Comment:
```suggestion
            -c "client state: MBEDTLS_SSL_CERTIFICATE_REQUEST" \
```

Reply:
I'd like do it in #5537 . That can reduce preceding PR of #5748 . 

With this change, #5748 have to precede #5537.

Reply:
This is a check on the client side that the client parsed successfully the EncryptedExtension message sent by the server. This should fit well in this PR, shouldn't it ?

Reply:
Sorry, I misunderstand.